specification/monitor/resource-manager/Microsoft.Insights/preview/2021-07-01-preview/privateLinkScopes_API.json

```diff
@@ -541,13 +614,16 @@
           "200": {
             "description": "Successfully retrieved private endpoint connections.",
             "schema": {
-              "$ref": "#/definitions/PrivateEndpointConnectionListResult"
+              "$ref": "#/definitions/ExtendedPrivateEndpointConnectionListResult"
```

Comment:
Why changed from `PrivateEndpointConnectionListResult` to  `ExtendedPrivateEndpointConnectionListResult`

Change type name will be breaking changes and would require PR to go through breaking changes review board.

Reply:
we are extending the common PrivateEndpointConnection type and adding 2 properties. 
Keeping the old PrivateEndpointConnection name will cause confusion with common type here:
"../../../../../common-types/resource-management/v2/privatelinks.json"

Do you suggest just cloning the definition in the common type and thus keep the old name?

Reply:
Sorry, I am not able to suggestion what is the best solution, as there is too much changes in the PR (and many of them involves renaming model, or replace some model with another model from common-types json), and I am really not able to tell which change causes breaks, which does not.